samza-operator/src/main/java/org/apache/samza/operators/spec/OperatorSpecs.java

```diff
@@ -68,18 +67,21 @@ private static String getOperatorId() {
     return new SinkOperatorSpec<>(sinkFn);
   }
 
+
   /**
    * Creates a {@link WindowOperatorSpec}.
    *
-   * @param windowFn  the {@link WindowFn} function
-   * @param <M>  type of input {@link MessageEnvelope}
-   * @param <WK>  type of window key
-   * @param <WS>  type of {@link WindowState}
-   * @param <WM>  type of output {@link WindowOutput}
+   * @param windowFn the definition for this window that specifies everything about this {@link org.apache.samza.operators.windows.WindowFunction}.
```

Comment:
Delete "that species everything about this WindowFunction". If you keep this, maybe "the WindowFunction that specifies this Window."?